presto-orc/src/main/java/com/facebook/presto/orc/ByteArrayUtils.java

```diff
@@ -83,4 +83,20 @@ private static long longBytesToLong(long bytes)
     {
         return Long.reverseBytes(bytes) ^ Long.MIN_VALUE;
     }
+
+    public static long hash(byte[] bytes, int offset, int length)
+    {
+        // Constant from MurMur hash.
+        final long m = 0xc6a4a7935bd1e995L;
```

Comment:
We don't use final for local variables. This needs to be private static final member variable.